File: hw/viteri/u2/HW18ZooInPython/Zoo/model/Bird.py
from datetime import date
from model.Animal import Animal
from model.Cage import Cage

class Bird(Animal):
    def __init__(self, id, description, born_on_date, gender, cage, number_of_bones, vertebrate):
        super().__init__(id, description, born_on_date, gender, cage, number_of_bones, vertebrate)

File: ws/gualotuna/u2/WS24Zoo/Zoo/src/main/java/ec/edu/espe/zoo/model/Bird.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Brayan Gualotuña, Dev Dynasty, DCCO-ESPE
 */
public abstract class Bird extends Animal{
    
    public Bird(int id, String description, Date bornOnDate, char gender, Cage cage, int numberOfBones, int numberOfLegs) {
        super(id, description, bornOnDate, gender, cage, numberOfBones, numberOfLegs);
    }
    
   
}
